### ISSUE TITLE
FBT003 and `pytest.param`

### ISSUE THREAD

**Issue by jamesbraza** [charliermarsh/ruff#4172]
[`pytest.param`](https://docs.pytest.org/en/6.2.x/reference.html#pytest.param) is designed for `pytest.mark.parametrize`.

```python
import pytest

@pytest.mark.parametrize(
    ("arg1", "arg2"),
    [pytest.param("spam", False), pytest.param("ham", True)],
)
def test_foo(arg1: str, arg2: bool) -> None:  # noqa: FBT001
    pass
```

Running `ruff==0.0.263` on this:

```none
a.py:5:27: FBT003 Boolean positional value in function call
a.py:5:55: FBT003 Boolean positional value in function call
```

Running `flake8==6.0.0` with `flake8-boolean-trap==1.0.0` on this, there is no output.  Somehow, `ruff` seems to have a more comprehensive `FBT003` checker.

`pytest.param` doesn't support kwargs, so one has to use boolean positionals.  Any thoughts on filtering out `FBT003` for `pytest.param`?

**Comment by charliermarsh** [comment]
I don't mind filtering it out, I think we already have an allowlist for these.
